chapters/05-summary.Rmd

```diff
@@ -1,3 +1,23 @@
 # Summary {#summary}
 
-In summary we can conclude that...
\ No newline at end of file
+The benchmarking has shown that the new implementation can outperform the de-facto state of the art library 'KDEpy' in terms of runtime.
+
+In terms of accuracy it is slightly better in most cases and in general comparable to KDEpy.
+
+Additionally it has the benefit of allowing any distribution of the loc-scale family as kernel implemented in TensorFlow Probability[@googleTFP]. At the moment this includes twelve kernels, namely `Cauchy`, `DoublesidedMaxwell`, `Gumbel`, `Laplace`, `LogLogistic`, `LogNormal`, `Logistic`, `LogitNormal`, `Moyal`, `Normal`, `PoissonLogNormalQuadratureCompound`, `SinhArcsinh`.
+
+The proposed implementation restricts itself to the one-dimensional case. So far only KDEpy and Statsmodels implement a multidimensional KDE.
+Generalization to higher dimensionel kernel density estimation is however relatively straightforward to implement, since most TensorFlow API's are dimensionally agnostic.
```

Comment:
[8] That's not quite true! What do you mean by "dimensionally agnostic"? And, importantly, the Kernel has to be multidimensional as well!

If you mention it (and yes, this is good also for the future work section or something at the end), make it more explicit. It can also be somewhere where we motivate why we "only" look at the 1D case